README.md

```diff
@@ -379,3 +379,10 @@ The following files will be removed:
 * `[0-9]*/`
 
 For a `core = unittest` test suite this option does not change any behaviour.
+
+### Used By
+
+- [Tarantool](https://github.com/tarantool/tarantool) - is an in-memory database and application server
+- [tarantool-python](https://github.com/tarantool/tarantool-python) - Python driver for Tarantool
```

Comment:
It does not anymore: https://github.com/tarantool/tarantool-python/pull/189

That was just one test case, so don't count it.

Reply:
I applied the change myself to don't bother you. Hope you don't mind.